apps/admin/frontend/src/screens/printed_ballots_report_screen.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { electionMinimalExhaustiveSampleDefinition } from '@votingworks/fixtures';
+import React from 'react';
+import { fakeKiosk, fakeUsbDrive } from '@votingworks/test-utils';
+import { renderInAppContext } from '../../test/render_in_app_context';
+import { PrintedBallotsReportScreen } from './printed_ballots_report_screen';
+import { mockUsbDrive } from '../../test/helpers/mock_usb_drive';
+import { fakeFileWriter } from '../../test/helpers/fake_file_writer';
+
+beforeEach(() => {
+  const mockKiosk = fakeKiosk();
+  mockKiosk.getUsbDriveInfo.mockResolvedValue([fakeUsbDrive()]);
+  const fileWriter = fakeFileWriter();
+  mockKiosk.saveAs = jest.fn().mockResolvedValue(fileWriter);
+  mockKiosk.writeFile = jest.fn().mockResolvedValue(fileWriter);
+  window.kiosk = mockKiosk;
+});
```

Comment:
If you need to decrease the code coverage threshold in the Jest config, feel free to (that's why CI is failing - the threshold is no longer being met). I know you mentioned at standup the conscious decision not to test all the internals of PDF rendering right now